### ISSUE TITLE
improve dock tabs look-and-feel

### ISSUE THREAD

**Issue by sbourdeauducq** [pyqtgraph/pyqtgraph#218]
The look and feel of tabs in applications like Google Chrome or Sublime Text is better IMO. Drag-and-drop of tabs to change their order should also be supported.

Also, they should follow the user's color scheme - the hardcoded blue colors do not always look the best.

**Comment by j9ac9k** [comment]
I'm going to close out this issue as it's not quite descriptive enough.  Our docks are customizable, and I will admit the defaults aren't great.  We would certainly welcome a PR to make them look more modern.

**Comment by sem-geologist** [comment]
I  think what @sbourdeauducq meant with "tabs" is actually `DockLabels`. Colors shapes and border color is hardcoded which makes it look acceptable in some Qt Themes, and in other it looks completely out of place. Currently I am preparing a Poster to promote my software which uses `pyqtgraph` and found that Docking tab color  bites with the rest of whole theme (It had not bothered me so much to use everyday, but making presentation for other people made me realize that), and so I dived into the code to find out if this can't be somehow changed. As I want to be ready also to show that my software also can be dark themed or light themed, I looked if colors could not be looked up and picked up dynamically from `.palette()`. It indeed could to do that easily as DockLabel class has that getter inherited!

The Modern look - To be honest I am not sure what that is. It has round corners - some would say that is modern other would say that it is already _passe_ - it is kind of taste thing. As I am used to DarkTheme Breeze KDE Neon or Light Breeze themes, I would move the border line to top and make it the highlighting element, where background and foreground would pick that from `.pallete().color()` with `QPalette.Window` and `QPalette.WindowText` enumerators.

I could try opening PR with such improvements. But then why it was not done like that from the beginning? maybe @campagnola wanted it to stand-out like this? I think blue colors used in pytgraph.org webpage is with similar hue, and InfiniteLine range or other default colors are also somehow connected.

Now draggable tabs are not so easy to achieve. We should remember that this pyqtgraph implemented Docking Framework is minimalistic - and that I think is the point.
If all bells and whistles are needed then there are something like this (a full blown library dedicated for docking and tabs) https://github.com/githubuser0xFFFF/Qt-Advanced-Docking-System , which actually seems to have at last python bindings installable with pip.
